ws-client/src/client.rs

```diff
@@ -102,14 +102,34 @@ pub struct WsClient {
 	id_guard: RequestIdGuard,
 }
 
-/// Configuration.
+/// Builder for [`WsClient`].
+///
+/// # Examples
+///
+/// no_run```
```

Comment:
Shouldn't this be `'''no_run` (but with actual backticks)? It would be better if it was actually compiled though.